In-house patch to remove the hardcoding of /dev/log as the address for
syslog.  The default logging module is smart enough to use the proper
path in Solaris.

--- oslo.log-1.0.0/oslo_log/log.py.orig	2015-09-29 16:41:03.335483630 -0600
+++ oslo.log-1.0.0/oslo_log/log.py	2015-09-29 16:41:23.112003925 -0600
@@ -285,11 +285,9 @@ def _setup_logging_from_conf(conf, proje
             # TODO(bogdando) use the format provided by RFCSysLogHandler
             #   after existing syslog format deprecation in J
             if conf.use_syslog_rfc_format:
-                syslog = handlers.RFCSysLogHandler(address='/dev/log',
-                                                   facility=facility)
+                syslog = handlers.RFCSysLogHandler(facility=facility)
             else:
-                syslog = logging.handlers.SysLogHandler(address='/dev/log',
-                                                        facility=facility)
+                syslog = logging.handlers.SysLogHandler(facility=facility)
             log_root.addHandler(syslog)
         except socket.error:
             log_root.error('Unable to add syslog handler. Verify that syslog '
